src/datamigration/setup.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup, find_packages
+
+VERSION = "0.2.0"
+
+CLASSIFIERS = [
+    'Development Status :: 4 - Beta',
+    'Intended Audience :: Developers',
+    'Intended Audience :: System Administrators',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 2',
+    'Programming Language :: Python :: 2.7',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.4',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: 3.6',
+    'License :: OSI Approved :: MIT License',
+]
+
+DEPENDENCIES = []
+
+setup(
+    name='dms-pg-mysql',
+    version=VERSION,
+    description='Microsoft Azure Command-Line Tools for the Data Migration Service (DMS) Command Module',
```

Comment:
Summary should not include references such as `command-line`, `CLI`, `extension`, etc as per https://github.com/Azure/azure-cli-extensions/blob/master/docs/extension_summary_guidelines.md.
These are all implicitly implied.